fidget/src/core/vm/alloc.rs

```diff
@@ -125,9 +125,9 @@ impl RegisterAllocator {
     /// Memory is treated as unlimited; if we don't have any spare slots, then
     /// we'll assign a new one (incrementing `self.total_slots`).
     ///
-    /// > If there's one thing I love  
-    /// > It's an infinite resource  
-    /// > If there's one thing worth loving  
+    /// > If there's one thing I love
```

Comment:
The trailing spaces are intentional here – these are [song lyrics](https://www.youtube.com/watch?v=Mk4oorsHTE0), and ending a line of Markdown with two spaces gives a hard line break when rendered.

Reply:
I see. That's just my editor stripping trailing spaces on save, I think.

Sorry about that. Maybe just use a `<br>` as that's valid Markdown but resistant to editor-cleaning-up-whitespace 'attacks'. :)